test-gui-e2e-setup.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -exuo pipefail
+
+###########################
+# Set up end to end tests #
+###########################
+# 1. Define and export ARTIFACT_DIR
+#
+# 2. Run the build script
+#
+# 3. Provide kubeadmin password in:
+#      /tmp/artifacts/installer/auth/kubeadmin-password
+#
+# 4. Log into console as kubeadmin
+#
+# 5. Set up test environment using oc-environment script
+
+
+ARTIFACT_DIR=/tmp/artifacts
+export ARTIFACT_DIR
+
+./build.sh
+
+oc login -u kubeadmin -p $(cat ${ARTIFACT_DIR}/installer/auth/kubeadmin-password)
```

Comment:
Not strictly required since we set ARTIFACT_DIR above, but it's good practice to use double quotes here. Then if ARTIFACT_DIR is ever changed to have a space, things still work.

```suggestion
oc login -u kubeadmin -p $(cat "${ARTIFACT_DIR}/installer/auth/kubeadmin-password")
```